lib/uring/uring_stubs.c

```diff
@@ -151,17 +151,28 @@ ocaml_uring_sq_ready(value v_uring) {
 }
 
 void /* noalloc */
-ocaml_uring_set_timespec(value v_sketch_ptr, value v_timeout)
+ocaml_uring_set_timespec_native(value v_sketch_ptr, int64_t timeout)
 {
   struct __kernel_timespec *t = Sketch_ptr_val(v_sketch_ptr);
   t->tv_sec = 0;
-  t->tv_nsec = Int64_val(v_timeout);
+  t->tv_nsec = timeout;
+}
+
+void
+ocaml_uring_set_timespec(value v_sketch_ptr, value v_timeout)
+{
+  return ocaml_uring_set_timespec_native(v_sketch_ptr, Int64_val(v_timeout));
 }
 
 #define Val_boottime Val_int(0)
 
 value /* noalloc */
-ocaml_uring_submit_timeout(value v_uring, value v_id, value v_sketch_ptr, value v_clock, value v_absolute)
+ocaml_uring_submit_timeout_native(value v_uring,
+    value v_id,
+    value v_sketch_ptr,
+    value v_clock,
+    value v_absolute,
+    int completion_count)
```

Comment:
The docs https://v2.ocaml.org/manual/intfc.html#s%3AC-cheaper-call say:
> Note: do not use the C int type in correspondence with `(int [@untagged])`. This is because they often differ in size.

But I don't think this should be an `int` anyway, as there's no untagged attribute (note that the MDX tests are run in byte-code mode).

Reply:
Good catch. This is fixed now.